modules/monitoring-querying-metrics-by-using-the-federation-endpoint-for-prometheus.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * monitoring/accessing-third-party-monitoring-apis.adoc
+
+:_content-type: PROCEDURE
+[id="monitoring-querying-metrics-by-using-the-federation-endpoint-for-prometheus_{context}"]
+= Querying metrics by using the federation endpoint for Prometheus
+
+You can use federation to query a cluster's scraped metrics from a network location outside the cluster. 
+To do so, access the Prometheus `/federate` endpoint for the cluster via an {product-title} route. 
+
+You can access the `/federate` endpoints for default platform monitoring and for user-defined projects.
+
+[WARNING]
+====
+A delay in retrieving metrics data occurs when you use federation. 
+This delay can affect the accuracy and timeliness of the scraped metrics.
+
+In addition, federation can degrade the performance and scalability of your cluster, especially if you use federation to try and retrieve large amounts of metrics data.
+
+For these reasons, use federation only when you want to retrieve a limited, aggregated data set.
+Do not try to retrieve all metrics data via the Prometheus federation API.
+
+If you need to forward large amounts of data outside the cluster, use remote write instead, if possible. See the _Configuring remote write storage_ section for more information.
+====
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+* You have access to the cluster as a user with the `cluster-monitoring-view` role.
```

Comment:
user with the `cluster-monitoring-view` role is not able to get the prometheus federation route

Reply:
we can add that knowledge of the federation URL is a prerequisite.